visual-recognition/v3-generated.ts

```diff
@@ -611,7 +610,6 @@ namespace VisualRecognitionV3 {
   export type Options = {
     version: string;
     url?: string;
-    api_key?: string;
```

Comment:
Do we need to keep this to maintain compatibility until next major release?

Reply:
We shouldn't need to since it is available in the base class options but I will confirm

Reply:
Confirmed, this still transpiles in TypeScript.